website_form_recaptcha/static/src/js/field_recaptcha.js

```diff
@@ -7,6 +7,7 @@ odoo.define('website_form_recaptcha.recaptcha', function (require) {
 
     var ajax = require('web.ajax');
     var snippet_animation = require('website.content.snippets.animation');
+    require("website_form.animation");
```

Comment:
Don't you assign it to any variable?

Reply:
it's just declaring a dependency and in any case it does not expose anything (unfortunately).

Reply:
No, this module just adds an animation to the registry. We get it from there below.